### PR TITLE
fix(carousel): fixes carousel warning

### PR DESCRIPTION
**Pull request by flesher1**
moves arrow buttons into outside classes. Removes the console warning
relating to carousel.

Adds a few extra lines of code but gets rid of the console warning about carousel. 
I made sure to maintain all the current functionality (including aria).

### CODE REVIEW

**Comment by irtiza-rizvi** [comment]
It's nice to not have that error in the console logs.  This also needs to be changed in `ListingOverview`.

File: src/components/listing/ListingOverview.js [hunk 1]
```diff
@@ -9,6 +9,30 @@ import { ShortListing } from 'services/serializers/listing';
 import Lightbox from 'react-image-lightbox';
 import moment from 'moment';
 
+function NextArrow(props) {
```
Comment (irtiza-rizvi) [inline src/components/listing/ListingOverview.js]: I would recommend putting `NextArrow` and `PrevArrow` in a file in the shared components folder and `export` them for use across components.

**Comment by irtiza-rizvi** [comment]
Minor comment.  Overall, functions well.  :+1:

**Comment by blynch11p** [comment]
:+1:

**Pull request merged**